rover_msgs/FuseData.lcm

```diff
@@ -0,0 +1,11 @@
+package rover_msgs;
+
+struct FuseData {
+    // Each array:
```

Comment:
This LCM looks fine. I haven't really investigated the fuse board traces to see which sensors (current data + voltage data) map to which devices (cytrons, nichrome wires, or h-bridges), but I think that's an ESW problem so we can keep to this convention (listed in the comments) for the lcms.

Reply:
can you guys add units to the LCMs? like current_amps (or what'll be more readable to display is  mA), voltage_volts, etc